pipelines/package_names.go

```diff
@@ -39,6 +39,7 @@ func TarFilename(opts TarFileOpts) string {
 }
 
 func TarOptsFromFileName(filename string) TarFileOpts {
+	filename = filepath.Base(filename)
```

Comment:
you may want to merge with main on this, this function had some improvements to handle non-tar.gz extensions and might affect your changes.

Reply:
oh nvm I see you did already oops